### PR TITLE
Junit5 itest refactor

### PR DESCRIPTION
**Pull request by ericenns**
## Description of changes
Refactored IntegrationTests to start an instance of the application directly, instead of relying on `spring-boot-maven-plugin` (Note: `pre-integration-test` execution config is still required by `springdoc-openapi-maven-plugin` which is why the execution was left in the pom file). Removing this reliance has simplified arguments for integration testing which can be seen in `./run-tests.sh`.

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

~* [ ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.~
~* [ ] Tests added (or description of how to test) for any new features.~
* [x] User documentation updated for UI or technical changes.

### CODE REVIEW

File: src/test/java/ca/corefacility/bioinformatics/irida/config/IridaIntegrationTestUriConfig.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import org.springframework.context.annotation.Profile;
+import org.springframework.context.event.EventListener;
+
+import ca.corefacility.bioinformatics.irida.ria.integration.pages.AbstractPage;
+import ca.corefacility.bioinformatics.irida.ria.integration.utilities.RemoteApiUtilities;
+import ca.corefacility.bioinformatics.irida.ria.web.oauth.OltuAuthorizationController;
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.http.ContentType;
+
```
Comment (deepsidhu85) [inline src/test/java/ca/corefacility/bioinformatics/irida/config/IridaIntegrationTestUriConfig.java:18]: Add a file comment for what this is used for

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
Just the one change. Looks good otherwise

File: src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/utilities/RemoteApiUtilities.java [hunk 2]
```diff
@@ -8,9 +8,13 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class RemoteApiUtilities {
-	protected static final String BASE_URL =
+	protected static String BASE_URL =
```
Comment (joshsadam) [inline src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/utilities/RemoteApiUtilities.java:11]: Are you going to remove this?

File: src/test/java/ca/corefacility/bioinformatics/irida/config/IridaIntegrationTestUriConfig.java (hunk 1 above)
Comment (ericenns) [reply src/test/java/ca/corefacility/bioinformatics/irida/config/IridaIntegrationTestUriConfig.java:18]: Addressed in dc4974a4ee

**Review by joshsadam — APPROVED** [review]
Looks awesome

**Review by deepsidhu85 — APPROVED** [review]

**Pull request merged**